W0903 23:18:01.546000 17531 site-packages/torch/utils/cpp_extension.py:2425] TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
W0903 23:18:01.546000 17531 site-packages/torch/utils/cpp_extension.py:2425] If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'] to specific architectures.
/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/torch/cuda/__init__.py:283: UserWarning: 
    Found GPU0 Tesla P100-PCIE-16GB which is of cuda capability 6.0.
    Minimum and Maximum cuda capability supported by this version of PyTorch is
    (7.0) - (12.0)
    
  warnings.warn(
/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/torch/cuda/__init__.py:304: UserWarning: 
    Please install PyTorch with a following CUDA
    configurations:  12.6 following instructions at
    https://pytorch.org/get-started/locally/
    
  warnings.warn(matched_cuda_warn.format(matched_arches))
/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/torch/cuda/__init__.py:326: UserWarning: 
Tesla P100-PCIE-16GB with CUDA capability sm_60 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_70 sm_75 sm_80 sm_86 sm_90 sm_100 sm_120.
If you want to use the Tesla P100-PCIE-16GB GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(
Jitting Chamfer 3D
Global seed set to 42
/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/pytorch_lightning/utilities/device_parser.py:135: LightningDeprecationWarning: Parsing of the Trainer argument gpus='0' (string) will change in the future. In the current version of Lightning, this will select CUDA device with index 0, but from v1.5 it will select gpus [] (same as gpus=0 (int)).
  rank_zero_deprecation(
/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/pytorch_lightning/trainer/connectors/accelerator_connector.py:110: LightningDeprecationWarning: `Trainer(distributed_backend=dp)` has been deprecated and will be removed in v1.5. Use `Trainer(accelerator=dp)` instead.
  rank_zero_deprecation(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
Loaded JIT 3D CUDA chamfer distance
dataset:  surreal

Log directory:
/home/du2/22CS30065/SE-ORNet/output/shape_corr/PointCorrWithAngle/arch_PointCorrWithAngle/dataset_name_surreal/latent_dim_768/03_09:23:19:00




/home/du2/22CS30065/SE-ORNet/output/shape_corr/PointCorrWithAngle/arch_PointCorrWithAngle/dataset_name_surreal/latent_dim_768/03_09:23:19:00



Traceback (most recent call last):
  File "/home/du2/22CS30065/SE-ORNet/train.py", line 109, in <module>
    main_train(model_class_pointer, hparams, parser)
  File "/home/du2/22CS30065/SE-ORNet/train.py", line 81, in main_train
    trainer.fit(model)
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 553, in fit
    self._run(model)
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 865, in _run
    self._call_setup_hook(model)  # allow user to setup lightning_module in accelerator environment
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 1171, in _call_setup_hook
    model.setup(stage=fn)
  File "/home/du2/22CS30065/SE-ORNet/models/shape_corr_trainer.py", line 46, in setup
    ) = switch_functions.load_dataset(self.hparams)
  File "/home/du2/22CS30065/SE-ORNet/utils/switch_functions.py", line 91, in load_dataset
    train_dataset = surreal(params,"train")
  File "/home/du2/22CS30065/SE-ORNet/data/point_cloud_db/surreal.py", line 73, in __init__
    super(surreal, self).__init__(params,split)
  File "/home/du2/22CS30065/SE-ORNet/data/point_cloud_db/point_cloud_dataset.py", line 57, in __init__
    self.verts,self.faces,self.d_max, self.gt_map = self.load_data(self.data_root, split,self.hparams)
  File "/home/du2/22CS30065/SE-ORNet/data/point_cloud_db/surreal.py", line 92, in load_data
    surreal_faces = trimesh.load("./data/surreal_template.ply", process=False).faces
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/trimesh/exchange/load.py", line 111, in load
    loaded = load_scene(
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/trimesh/exchange/load.py", line 193, in load_scene
    arg = _parse_file_args(
  File "/home/du2/22CS30065/miniconda3/envs/se_ornet/lib/python3.10/site-packages/trimesh/exchange/load.py", line 624, in _parse_file_args
    raise ValueError(f"string is not a file: `{file_obj}`")
ValueError: string is not a file: `./data/surreal_template.ply`
